cves/kernel/CVE-2018-20169.yml

```diff
@@ -339,10 +266,53 @@ stacktrace:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  any_stacktraces:
-  stacktrace_with_fix:
-  note:
+  any_stacktraces: false
+  stacktrace_with_fix: false
+description: The USB subsystem fails to check size of extra descriptor by the
+  minimum and maximum size allowed, to prevent from invalid data being sent by a
+  USB device and crashing the host device.
+unit_tested:
+  fix: false
+  code: false
+  question: |
+    Were automated unit tests involved in this vulnerability?
+    Was the original code unit tested, or not unit tested? Did the fix involve
+    improving the automated tests?
+
+    For code: and fix: - your answer should be boolean.
+
+    For the code_answer below, look not only at the fix but the surrounding
+    code near the fix in related directories and determine if and was there were
+    unit tests involved for this subsystem.
+
+    For the fix_answer below, check if the fix for the vulnerability involves
+    adding or improving an automated test to ensure this doesn't happen again.
+  fix_answer:
+  code_answer:
+reported_date: 2018-12-05
+specification:
+  note: The USB protocol for extra descriptor size is violated
```

Comment:
this is good, but do you think you could also add links to this resource where you found that it was violated?

Reply:
The specification itself was not violated, but the bug was triggered by a device that violates the specification, reworded to improve clarity